interfaces/builtin/opengl.go

```diff
@@ -65,6 +65,15 @@ const openglConnectedPlugAppArmor = `
   /run/udev/data/c226:[0-9]* r,  # 226 drm
 `
 
+const openglConnectedPlugUdev = `
+# This file contains udev rules for opengl drivers.
+#
+# Do not edit this file, it will be overwritten on updates
+
+KERNEL=="card[0-9]*",         NAME="dri/card%%n", TAG+="%[1]s"
+KERNEL=="nvidia*|nvidiactl*", NAME="%%k",         TAG+="%[1]s"
```

Comment:
I'm not terribly familiar with this device. I see these:
- https://bugs.launchpad.net/snappy/+bug/1500755
- https://bugs.launchpad.net/snapd/+bug/1533265
- https://raspberrypi.stackexchange.com/questions/54571/what-is-dev-vchiq-in-raspberry-pi